lib/setup.js

```diff
@@ -236,7 +236,7 @@ export default function (app) {
     const isProduction = NODE_ENV === "production"
     const code = req.timedout ? 504 : (err.status || 500)
     const message = !isProduction ? err.message || err.text || err.toString() : ""
-    const detail = !isProduction ? err.stack && !isProduction : ""
+    const detail = !isProduction ? err.stack : ""
```

Comment:
Oh yea good call. I'll do that and try to move it into a middleware and a basic spec.